pkg/storage/mongo/test_suite.go

```diff
@@ -33,12 +33,12 @@ func newTestStore() (*ModuleStore, error) {
 		return nil, err
 	}
 
-	mongoStore := NewStorage(muri)
+	mongoStore, err := NewStorage(muri)
 	if mongoStore == nil {
```

Comment:
Maybe just check the error instead?

Reply:
+1, I feel like that would read nicer. @kteb can you do an `if err != nil` here instead?